Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -751,14 +751,15 @@ extension PlaceDetailViewController: InAppNotificationToastDelegate {
 }
 
 extension PlaceDetailViewController: FilterViewControllerDelegate {
-    func filterViewController(_ filterViewController: FilterViewController, didUpdateFilters filters: [PlaceFilter]) {
-        guard let count = dataSource?.filterPlaces(filters: filters).count else { return }
+    func filterViewController(_ filterViewController: FilterViewController, didUpdateFilters filters: [PlaceFilter], topRatedOnly: Bool) {
+        guard let count = dataSource?.filterPlaces(filters: filters, topRatedOnly: topRatedOnly).count else { return }
         filterViewController.placeCount = count
     }
 
-    func filterViewController(_ filterViewController: FilterViewController, didDismissWithFilters viewFilters: [PlaceFilter]) {
+    func filterViewController(_ filterViewController: FilterViewController, didDismissWithFilters viewFilters: [PlaceFilter], topRatedOnly: Bool) {
         guard let dataSource = dataSource else { return }
         dataSource.filters.enumerated().forEach { $1.enabled = viewFilters[$0].enabled }
+        dataSource.topRatedOnly = topRatedOnly
         dataSource.refresh()
```

Comment:
I think I'd prefer:
```swift
class PlacesProvider {
    private(set) var topRatedOnly = ...
    func refresh(topRatedOnly: Boolean? = nil) {
        if let topRatedOnly != nil { self.topRatedOnly = topRatedOnly }
        ...
    }
```
This is more atomic – I'm concerned of the inconsistency where `topRatedOnly` was updated (e.g. in `didDismissWithFilters` but we forgot to call `refresh` so `topRatedOnly` is no longer consistent with the underlying data.